azure-pipelines.yml

```diff
@@ -0,0 +1,81 @@
+name: 3.0.0-ci.$(Date:yyyyMMdd).$(Rev:r) # equal to $(Build.BuildNumber)
+trigger:
```

Comment:
Any reason to limit the branches built to this list, shouldn't we just build all branches?